src/shared/transaction-item/transaction-item.html

```diff
@@ -20,10 +20,12 @@
 	<span class="btn-copy" clip-copy="tx.id"></span>
 </td>
 <td data-title="Sender">
-	<a data-account-href="tx" data-type="sender"><span>{{tx | txSender}}</span></a>
+	<a data-ng-if="tx.senderId === address" data-type="sender" class="disabled">{{tx | txSender}}</a>
```

Comment:
Why not use a different html tag rather than adding the `disabled` class to the `a` tag?
It could be `span` for instance.

Reply:
I've tried with `<p>` first, but `<a>` has one significant advantage: CSS attributes, esp. spacings are already defined for that element, using `<p>` we would have to copy the whole definition except the font color and cursor shape.